entities/manager.rb

```diff
@@ -0,0 +1,156 @@
+class Manager
+  include Taxes
+  include Console
+  include DataUtils
+
+  OPERATIONS = {
+    show_cards: 'SC',
+    create_card: 'CC',
+    destroy_card: 'DC',
+    put_money: 'PM',
+    withdraw_money: 'WM',
+    destroy_account: 'DA'
+  }.freeze
+
+  attr_accessor :current_account
+
+  def initialize
+    @file_path = PATH
+    @current_account = nil
+  end
+
+  def console
+    case start
+    when 'create' then create
+    when 'load' then load
+    else exit
+    end
+  end
+
+  def create
+    name = name_input
+    age = age_input
+    login = login_input(load_accounts.map(&:login))
+    password = password_input
+    @current_account = Account.new(name: name, age: age, login: login, password: password)
+    save_account(@current_account, @file_path)
+    main_menu
+  end
+
+  def show_cards
+    cards_show(@current_account.card)
+  end
+
+  def operate_money(operation)
+    return puts I18n.t(:no_cards) if @current_account.card.none?
+
+    card = card_for_operation(operation)
+    return unless card
+
+    return calculate_withdraw_money(*items_for_operation(card, operation)) if operation == 'withdraw'
+
+    calculate_put_money(*items_for_operation(card, operation))
+  end
+
+  def card_for_operation(operation)
+    puts I18n.t(operation.to_sym)
+    card = choose_card(@current_account.card)
+    card || exit
```

Comment:
You can combine this line and line above to `choose_card(@current_account.card) || exit`